source/_integrations/light.modbus.markdown

```diff
@@ -0,0 +1,139 @@
+---
+title: "Modbus Light"
+description: "Instructions on how to integrate Modbus lights into Home Assistant."
+ha_category:
+  - Light
+ha_release: 0.108
+ha_iot_class: Local Push
+ha_domain: modbus
+---
+
+The `modbus` light platform allows you to control [Modbus](http://www.modbus.org/)-enabled lights. We supports two types of Modbus lights — coil-based and register-based. Note that our implementation of Modbus register light supports only holding registers, because input registers are not writable, and don't give us any control over connected lights.
```

Comment:
```suggestion
The `modbus` light platform allows you to control [Modbus](http://www.modbus.org/)-enabled lights. We support two types of Modbus lights — coil-based and register-based. Note that our implementation of Modbus register light supports only holding registers, because input registers are not writable, and don't give us any control over connected lights.
```